lib/Dialect/SV/SVAttributes.cpp

```diff
@@ -26,6 +26,18 @@ using namespace circt::sv;
 
 #include "circt/Dialect/SV/SVEnums.cpp.inc"
 
+llvm::StringRef svAttr = "sv.attributes";
```

Comment:
Seems like this should be in a header file somewhere. Perhaps in SVAttributeAttr?